### PR TITLE
Folder structure an ATTRIBUTION file

### PR DESCRIPTION
**Pull request by rouille**
[Pull Request doc](https://breakthrough-energy.github.io/docs/user/git_guide.html#d-pull-request)

### Purpose
Create folder structure and update **ATTRIBUTION.md**. HIFLD files have been uploaded to the BES blob storage.

### What the code is doing
N/A

### Testing
N/A

### Where to look
* Formatted a previous entry
* 3 new entries for HIFLD data (transmission, substation and generation)

### Usage Example/Visuals
N/A

### Time estimate
10min

### CODE REVIEW

File: ATTRIBUTION.md [hunk 1]
```diff
@@ -333,4 +333,63 @@ The dataset is used to generate wind profiles from wind speed profiles.
...
+* Modifications to source file(s): None
+* Location:
+  * https://besciences.blob.core.windows.net/datasets/hifld/Electric_Power_Transmission_Lines_Jul2020.csv
+  * https://besciences.blob.core.windows.net/datasets/hifld/Electric_Power_Transmission_Lines_Jul2020.geojson.zip
+
+##### General Purpose
+The data are used to populate the transmission topology of U.S. power grid that can be used to run scenarios.
+
+##### Note
+Public use (see https://hifld-geoplatform.opendata.arcgis.com/datasets/geoplatform::electric-power-transmission-lines/about). Users are advised to read the data set's metadata thoroughly to understand appropriate use and data limitations)
```
Comment (danielolsen) [inline ATTRIBUTION.md:357]: Extra `)`

File: ATTRIBUTION.md [hunk 2]
```diff
@@ -333,4 +333,63 @@ The dataset is used to generate wind profiles from wind speed profiles.
...
+
+##### Destination
+* Modifications to source file(s): None
+* Location: https://besciences.blob.core.windows.net/datasets/hifld/Electric_Substations_Jul2020.csv
+
+##### General Purpose
+The data are used to populate the substation topology of U.S. power grid that can be used to run scenarios.
+
+##### Note
+Public use (see https://hifld-geoplatform.opendata.arcgis.com/datasets/geoplatform::electric-substations/about). Users are advised to read the data set's metadata thoroughly to understand appropriate use and data limitations)
```
Comment (danielolsen) [inline ATTRIBUTION.md:375]: Extra `)`

File: ATTRIBUTION.md [hunk 3]
```diff
@@ -333,4 +333,63 @@ The dataset is used to generate wind profiles from wind speed profiles.
...
+
+##### Destination
+* Modifications to source file(s): None
+* Location: https://besciences.blob.core.windows.net/datasets/hifld/Power_Plants_Jul2020.csv
+
+##### General Purpose
+The data are used to populate the power generation profile of U.S. transmission network that can be used to run scenarios.
+
+##### Note
+Public use (see https://hifld-geoplatform.opendata.arcgis.com/datasets/geoplatform::power-plants/about). Users are advised to read the data set's metadata thoroughly to understand appropriate use and data limitations)
```
Comment (danielolsen) [inline ATTRIBUTION.md:393]: Extra `)`

File: ATTRIBUTION.md [hunk 4]
```diff
@@ -333,4 +333,63 @@ The dataset is used to generate wind profiles from wind speed profiles.
...
+The data are used to populate the transmission topology of U.S. power grid that can be used to run scenarios.
+
+##### Note
+Public use (see https://hifld-geoplatform.opendata.arcgis.com/datasets/geoplatform::electric-power-transmission-lines/about). Users are advised to read the data set's metadata thoroughly to understand appropriate use and data limitations.
+
+
+##### Source
+* Name: Electric Substations
+* Author: HIFLD Open
+* Description: This dataset represents electric power substations primarily associated with electric power transmission. In this layer, substations are considered facilities and equipment that switch, transform, or regulate electric power at voltages equal to, or greater than, 69 kilovolts.
```
Comment (BainanXia) [inline ATTRIBUTION.md:363]: Should we use kV instead of kilovolts to be consistent here.

**Review by BainanXia — COMMENTED** [review]
I presume other file changes are due to a rebase onto the latest `develop`.

File: ATTRIBUTION.md (hunk 1 above)
Comment (rouille) [reply ATTRIBUTION.md]: Done

File: ATTRIBUTION.md (hunk 2 above)
Comment (rouille) [reply ATTRIBUTION.md]: Done

File: ATTRIBUTION.md (hunk 3 above)
Comment (rouille) [reply ATTRIBUTION.md]: Done

File: ATTRIBUTION.md (hunk 4 above)
Comment (rouille) [reply ATTRIBUTION.md]: Done

**Review by danielolsen — APPROVED** [review]

**Comment by rouille** [comment]
> I presume other file changes are due to a rebase onto the latest `develop`.

Yeah, I rebased `hifld` onto the lates `develop` but forgot to force push

**Review by BainanXia — APPROVED** [review]
Looks good.

**Review by ahurli — APPROVED** [review]
Thanks!

**Pull request merged**